Sources/Plasma/NucleusLib/pnAsyncCoreExe/pnAceDns.cpp

```diff
@@ -125,7 +117,7 @@ static void AddressResolved(const asio::error_code& err,
     PerfSubCounter(kAsyncPerfNameLookupAttemptsCurr, 1);
 }
 
-void AsyncAddressLookupName(FAsyncLookupProc lookupProc,
+void AsyncAddressLookupName(FAsyncLookupProc  lookupProc,

```

Comment:
```suggestion
void AsyncAddressLookupName(FAsyncLookupProc lookupProc,

```

Looks like a bit of spurious whitespace here.